review/passenger-service/src/test/java/cz/muni/fi/airportmanager/passengerservice/repository/PassengerRepositoryTest.java

```diff
@@ -43,6 +59,8 @@ void shouldFindNotificationsForPassenger(UniAsserter asserter) {
     @Test
     @TestReactiveTransaction
     void shouldAddNotificationByFlightId(UniAsserter asserter) {
+        // TODO implement this test
+        // It should test that the notification is added to the appropriate passengers
```

Comment:
Will students know which methods they need to call? won't this be too hard to find in the code? The description is too general.